src/Shell/Task/LoadTask.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         3.0.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Shell\Task;
+
+use Cake\Console\Shell;
+use Cake\Filesystem\File;
+
+/**
+ * Task for loading plugins.
+ *
+ */
+class LoadTask extends Shell
+{
+    /**
+     * Path to the bootstrap file.
+     *
+     * @var string
+     */
+    public $bootstrap = null;
+
+    /**
+     * Execution method always used for tasks.
+     *
+     * @param string $plugin The plugin name.
+     * @return bool
+     */
+    public function main($plugin = null)
+    {
+        $this->bootstrap = ROOT . DS . 'config' . DS . 'bootstrap.php';
+
+        if (empty($plugin)) {
+            $this->err('<error>You must provide a plugin name in CamelCase format.</error>');
+            $this->err('To load an "Example" plugin, run <info>`cake plugin load Example`</info>.');
+            return false;
+        }
+
+        return (bool)$this->_modifyBootstrap($plugin, $this->params['bootstrap'], $this->params['routes'], false);
+    }
+
+    /**
+     * Update the applications bootstrap.php file.
+     *
+     * @param string $plugin Name of plugin.
+     * @param bool $hasBootstrap Whether or not bootstrap should be loaded.
+     * @param bool $hasRoutes Whether or not routes should be loaded.
+     * @param bool $hasAutoloader Whether or not there is an autoloader configured for
+     * the plugin.
+     * @return bool If modify passed.
+     */
+    protected function _modifyBootstrap($plugin, $hasBootstrap, $hasRoutes, $hasAutoloader)
```

Comment:
I have copied this part from the bake plugin. I let this so there are possibilities in future to extend this. I have not much time this week to fix stuff